ocs_ci/ocs/machine.py

```diff
@@ -187,6 +187,24 @@ def add_capacity(count, storagecluster_name, namespace=defaults.ROOK_CLUSTER_NAM
     ocp.exec_oc_cmd(cmd)
     return True
 
+def add_storage_capacity(capacity, storagecluster_name, namespace=defaults.ROOK_CLUSTER_NAMESPACE):
+    """
+    Add storage capacity to the cluster
+
+    Args:
+        capacity (str): Size of the storage
+        storagecluster_name (str): Name of a storage cluster
+    Returns:
+        bool: True if commands executes successfully
+    """
+    ocp = OCP(namespace=namespace)
+    # ToDo Update patch command with pr https://github.com/red-hat-storage/ocs-ci/pull/803
+    cmd = f'''
```

Comment:
@shreekarSS dont you have this function, If so I suggest we use only one and get rid of another.

Reply:
@fmarchioni I was talking about this one https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/cluster/cluster_expansion/test_add_capacity.py#L21

To avoid duplication, we can fix also fix the script to use this function.